### PR TITLE
IOPS 2486

### PR DESCRIPTION
**Pull request by vickyjaiswal0308**

### CODE REVIEW

File: examples/UKCore-Bundle-Batch-Example.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Bundle xmlns="http://hl7.org/fhir">
+ <id value="UKCore-Bundle-Batch-Example" />
+ <type value="batch" />
+ <entry>
+  <request>
+   <method value="GET" />
+   <url value="https://fhir.hl7.org.uk/Patient/example" />
+  </request>
+   <request>
```
Comment (APageNHS) [inline examples/UKCore-Bundle-Batch-Example.xml:9]: fix needs to be in irts own entry tag

File: examples/UKCore-Bundle-Batch-Example.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <method value="GET" />
+    <url value="https://fhir.hl7.org.uk/Condition?patient=example" />
+   </request>
+   <request>
```
Comment (APageNHS) [inline examples/UKCore-Bundle-Batch-Example.xml:13]: this needs to be in its own entry tag

**Comment by APageNHS** [comment]
Example is currently invalid (needs entry tags), and it for a batch get - can we also get a batch put example?

File: examples/UKCore-Bundle-Batch-Example.xml (hunk 1 above)
Comment (vickyjaiswal0308) [reply examples/UKCore-Bundle-Batch-Example.xml:9]: why can't we have multiple requests in a single entry as that's how it has been done in json and other example which i saw?
{
   "type" : "batch",
   "resourceType" : "Bundle",
   "entry" : [
      {
         "request" : {
            "method" : "GET",
            "url" : "CodeSystem/$lookup?system=http://loinc.org&code=23245-4"
         }
      },
      {
         "request" : {
            "method" : "GET",
            "url" : "CodeSystem/$lookup?system=http://snomed.info/sct&code=17311000168105"
         }
      }
   ]
}

Comment (vickyjaiswal0308) [reply examples/UKCore-Bundle-Batch-Example.xml:11]: i have done the change still I don't know FHIR Validation call is failing

File: examples/UKCore-Bundle-BatchPut-Example.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <entry>
+    <resource>
+      <Parameters>
+        <parameter>
+          <name value='system'/>
+          <valueUri value='http://loinc.org'/>
+        </parameter>
+        <parameter>
+          <name value='code'/>
+          <valueCode value='23245-4'/>
```
Comment (APageNHS) [inline examples/UKCore-Bundle-BatchPut-Example.xml:13]: We don't really use LOINC in the UK, so this is bad example

File: examples/UKCore-Bundle-BatchPut-Example.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  <entry>
+    <resource>
+      <Parameters>
+        <parameter>
+          <name value='system'/>
+          <valueUri value='http://snomed.info/sct'/>
+        </parameter>
+        <parameter>
+          <name value='code'/>
+          <valueCode value='17311000168105'/>
```
Comment (APageNHS) [inline examples/UKCore-Bundle-BatchPut-Example.xml:31]: Namespace 1000168, is the australia drugs one, so thisconcept wouldn't be found on a UK terminology server. A sad matter of fact is that I know thats the AMT for panadol...

File: examples/UKCore-Bundle-BatchPut-Example.xml
Comment (APageNHS) [inline examples/UKCore-Bundle-BatchPut-Example.xml:1]: I'd rework this, and do something like a vital signs batch of blood pressures taken ever 5 minutes for an hour?

File: examples/UKCore-Bundle-Batch-Example.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<Bundle xmlns="http://hl7.org/fhir">
+ <id value="UKCore-Bundle-Batch-Example" />
+ <type value="batch" />
+ <entry>
+  <request>
+   <method value="GET" />
+   <url value="https://fhir.hl7.org.uk/Patient/example" />
```
[the remark this answers is not in the document]
Comment (vickyjaiswal0308) [reply examples/UKCore-Bundle-Batch-Example.xml]: Okay!! I have done this change.

File: examples/UKCore-Bundle-BatchPut-Example.xml (hunk 3 above)
Comment (vickyjaiswal0308) [reply examples/UKCore-Bundle-BatchPut-Example.xml:13]: Actually , i have just post the example sample right now still have to work on it to make it Ukfied.

**Review by ryma2fhir — APPROVED** [review]

**Pull request merged**